plasmapy/physics/transport/tests/test_collisions.py

```diff
@@ -12,6 +12,7 @@
                                         coupling_parameter)
 from plasmapy.physics.transport.collisions import Spitzer_resistivity
 from plasmapy.utils import RelativityWarning, RelativityError, PhysicsWarning
+from plasmapy.utils import exceptions
```

Comment:
There is already a utils import on the line above this one. Just add `exceptions` to that list. Might need to use parentheses and vertical spacing.